### PR TITLE
Adyen: Add level 2 and 3 information

### PR DESCRIPTION
**Pull request by Sainterman**
Summary:
To be able to add level II and III information to Adyen transactions
the Aden gateway is updated
Created 2 methods: add_level_2_data and add_level_3_data
These methods are used by authorize method

Unit Test:
Finished in 0.032293 seconds.
103 tests, 532 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote Test:
Finished in 752.767143 seconds.
127 tests, 398 assertions, 11 failures, 10 errors, 0 pendings, 0 omissions, 0 notifications
83.4646% passed

### CODE REVIEW

**Comment by curiousepic** [comment]
@Sainterman looks like there is a Rubocop failure

File: test/remote/gateways/remote_adyen_test.rb [hunk 1]
```diff
@@ -1379,6 +1379,46 @@ def test_successful_authorize_with_sub_merchant_sub_seller_data
     assert_success response
   end
 
+  def test_successful_authorize_with_lvl_2_data
```
Comment (javierpedrozaing) [inline test/remote/gateways/remote_adyen_test.rb:1382]: 💬 
I prefer to use the word level instead 'lvl', and It's just matter of personal taste, looks better use words instead numbers in functions

**Review by javierpedrozaing — COMMENTED** [review]
Good job, just I left a comment for your consideration and double check rubocop

**Comment by curiousepic** [comment]
@sainterman @edgarv09 Code looks good to me, but a couple things: 
- there are some formatting complaints form the Rubocop linter
- there are some other strange failures in the CI Tests, but rebasing from master may help
- please squash all commits into one before asking for review (or at least before I merge it).

**Pull request closed without merging**